pkg/e2e/verify/samesite_cookie.go

```diff
@@ -52,3 +72,18 @@ func managedRoutes(h *helper.H, namespace string) (bool, error) {
 	}
 	return samesiteExists, nil
 }
+
+func getClusterVersion(h *helper.H) (*v1.ClusterVersion, int, error) {
```

Comment:
Would you consider adding this as a method to `helper.H` instead and exporting it? I could see other `osde2e` users wanting to be able to do this.

Reply:
Absolutely, that is a great idea. I will explore this.